File: com/bridgeLab/dataStructure/banking/entity/Customer.java
package com.bridgeLab.dataStructure.banking.entity;

public class Customer {
	
	String name;
	String accountNumber;
	double amount;
	double balance;
	String operation;
	public String getOperation() {
		return operation;
	}
	public void setOperation(String operation) {
		this.operation = operation;
	}
	int token;
	
	public int getToken() {
		return token;
	}
	public void setToken() {
		
		this.token = (int) (Math.random()*10000);
	}
	public double getBalance() {
		return balance;
	}
	public void setBalance(double balance) {
		this.balance = balance;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public double getAmount() {
		return amount;
	}
	public void setAmount(double amount) {
		this.amount = amount;
	}
	public String getAccountNumber() {
		return accountNumber;
	}
	public void setAccountNumber(String accountNumber) {
		this.accountNumber = accountNumber;
	}
	

}

File: Month1/com/bridgelab/DataStructure/Banking/Customer.py
class Customer:
    def __init__(self):
        self.account_no = 0
        self.customer_name = None
        self.amount = None
        self.operation = None

    def get_account_no(self):
        return self.account_no

    def set_account_no(self, account_no):
        self.account_no = account_no

    def get_customer_name(self):
        return self.customer_name

    def set_customer_name(self, customer_name):
        self.customer_name = customer_name

    def get_amount(self):
        return self.amount

    def set_amount(self, amount):
        self.amount = amount

    def get_operation(self):
        return self.operation

    def set_operation(self, operation):
        self.operation = operation

    def to_string(self):
        return str(self.get_account_no()) + " " + self.get_customer_name() + " " + str(self.get_amount())
